homeassistant/components/lyric/__init__.py

```diff
@@ -88,13 +91,23 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     client_id = hass.data[DOMAIN][CONF_CLIENT_ID]
     lyric = Lyric(client, client_id)
 
+    try:
```

Comment:
Does it work if you remove this whole block since the coordinator update will raise it?

Reply:
It should work without it as async_config_entry_first_refresh() should reraise ConfigEntryAuthFailed